### PR TITLE
MoesHouse Zigbee Thermostatic Radiator

### PR DESCRIPTION
**Pull request by kirovilya**
MoesHouse Zigbee Thermostatic Radiator Valve https://github.com/Koenkk/zigbee2mqtt/issues/3221
and improvements for Siterwell GS361 smart thermostatic radiator valve https://github.com/Koenkk/zigbee2mqtt/issues/3113

### CODE REVIEW

File: converters/fromZigbee.js [hunk 1]
```diff
@@ -237,27 +231,80 @@ const siterwellGs361 = (model, msg, publish, options, meta) => {
      */
 
     switch (dp) {
+    case 104: // 0x6800 window params
+        return {
+            window_detection_params: {
+                valve: data[0] ? 'ON' : 'OFF',
+                temp: data[1],
```
Comment (Koenkk) [inline converters/fromZigbee.js:238]: Can you change `temp` to `temperature`?

File: converters/fromZigbee.js [hunk 2]
```diff
@@ -237,27 +231,80 @@ const siterwellGs361 = (model, msg, publish, options, meta) => {
...
+        };
+    case 112: // set schedule for workdays [6,0,20,8,0,15,11,30,15,12,30,15,17,30,20,22,0,15]
+        // 6:00 - 20*, 8:00 - 15*, 11:30 - 15*, 12:30 - 15*, 17:30 - 20*, 22:00 - 15*
+        return {workdays: [
+            {hour: data[0], min: data[1], temp: data[2]},
+            {hour: data[3], min: data[4], temp: data[5]},
+            {hour: data[6], min: data[7], temp: data[8]},
+            {hour: data[9], min: data[10], temp: data[11]},
+            {hour: data[12], min: data[13], temp: data[14]},
+            {hour: data[15], min: data[16], temp: data[17]},
```
Comment (Koenkk) [inline converters/fromZigbee.js:250]: What does `min` mean? `minimum_temperature`?

File: converters/toZigbee.js [hunk 3]
```diff
@@ -2043,102 +2053,104 @@ const converters = {
...
+    },
+    tuya_thermostat_comfort_temp: {
+        key: ['comfort_temp'],
+        convertSet: async (entity, key, value, meta) => {
+            const payloadValue = utils.convertDecimalValueTo2ByteHexArray(value);
+            sendTuyaCommand(entity, 619, 0, [4, 0, 0, ...payloadValue]);
+        },
+    },
+    tuya_thermostat_eco_temp: {
+        key: ['eco_temp'],
```
Comment (Koenkk) [inline converters/toZigbee.js:2137]: Also here change all `temp` to `temperature`.

File: devices.js [hunk 4]
```diff
@@ -9093,11 +9120,31 @@ const devices = [
     },
     {
         zigbeeModel: ['vivi ZLight'],
-        model: 'LifeControl_RGB_Led',
+        model: 'MCLH-02',
```
Comment (Koenkk) [inline devices.js:9123]: Make sure to rebase this PR, these changes shouldn't be included here.

File: converters/fromZigbee.js (hunk 1 above)
Comment (kirovilya) [reply converters/fromZigbee.js]: ok

File: converters/fromZigbee.js (hunk 2 above)
Comment (kirovilya) [reply converters/fromZigbee.js]: minute

File: devices.js (hunk 4 above)
Comment (kirovilya) [reply devices.js]: ok

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**